src/promptflow/promptflow/_core/_errors.py

```diff
@@ -4,6 +4,12 @@
 from promptflow.exceptions import ErrorTarget, SystemErrorException, UserErrorException, ValidationException
 
 
+class UnexpectedError(SystemErrorException):
+    """Exception raised for normal errors that should not happen."""
```

Comment:
Exception raised for unexpected errors that should not occur under normal circumstances.

Reply:
Great, thanks~